contrib/local-environment/dex.yaml

```diff
@@ -1,15 +1,15 @@
 # This configuration is intended to be used with the docker-compose testing
 # environment.
-# This should configure Dex to run on port 4190 and provides a static login
-issuer: http://dex.localtest.me:4190/dex
+# This should configure Dex to run on port 4195 and provides a static login
+issuer: http://dex.localtest.me:4195/dex
```

Comment:
Lets keep the dex and docker-compose.yaml as they are and instead introduce a new "dex-pkce.yaml" and "docker-compose-pkce.yaml" for testing this feature and still preserving the examples file for classic client secrets